rotkehlchen/db/upgrades/v34_v35.py

```diff
@@ -373,7 +373,7 @@ def _add_defillama_to_oracles(cursor: 'DBCursor', setting_name: Literal['current
     oracle_list = default_oracles
     if price_oracles is not None:
         oracles = json.loads(price_oracles[0])
-        oracle_list = [oracle_cls.deserialize(oracle) for oracle in oracles]
+        oracle_list = [oracle_cls.deserialize(oracle) for oracle in oracles if oracle != 'saddle']  # noqa: E501  # saddle got removed in 1.30 but could be present in all versions
```

Comment:
Why? It's totally fine for saddle to be there in v35

Reply:
when we call deserialize saddle is not a valid option anymore